README.md

```diff
@@ -153,93 +186,92 @@ This can be used to represent all kinds of errors. It can be for errors that hap
 | <a name="error.code"></a>`error.code`  | Error code describing the error.  | keyword  |   |   |
 
 
-## <a name="event"></a> Event fields
+### <a name="event"></a> Event fields
 
-The event fields are used for context information about the data itself.
+The event fields are carry context information about the data itself.
```

Comment:
The event fields carry context?

Reply:
Tried different wording.  Can change back to original if preferred.